string[] paths
---
kineverse/OperationsUpdate history